main/adwaita-icon-theme/APKBUILD

```diff
@@ -1,23 +1,18 @@
 # Contributor: Łukasz Jendrysik <scadu@yandex.com>
-# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
+# Contributor: Natanael Copa <ncopa@alpinelinux.org>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=adwaita-icon-theme
-pkgver=3.31.1
+pkgver=3.32.0
```

Comment:
```suggestion
pkgver=3.32.0
_major="${pkgver%${pkgver##[0-9]}}"
```

Reply:
Hm, I don't think this is necessary. They usually don't make `x.x.x.x` releases, that only happened once due to fix a minor packaging bug.